.github/workflows/build_workflow.yml

```diff
@@ -77,12 +77,12 @@ jobs:
 
       - if: ${{ steps.skip_check.outputs.should_skip != 'true' }}
         name: Set up Conda Environment
-        uses: conda-incubator/setup-miniconda@v2
+        uses: conda-incubator/setup-miniconda@v3
         with:
           activate-environment: "e3sm_diags_ci"
+          environment-file: conda-env/ci.yml
           miniforge-variant: Miniforge3
           miniforge-version: latest
-          environment-file: conda-env/ci.yml
```

Comment:
The build workflow was breaking with `setup-miniconda@v2`.

Example: https://github.com/E3SM-Project/e3sm_diags/actions/runs/14765144533/job/41454861424?pr=972